### DISCUSSION TITLE
When selecting a collection in `now playing`, other songs appear

### DISCUSSION THREAD

**Discussion by agatemosu** [ppy/osu#22991] [Q&A] [answered]
### Type

Other

### Bug description

If I select a collection in the `now playing` overlay, other songs that are not in the collection appear

### Version

2023.326.1-lazer

### Logs

not required

**Comment by bdach** [comment]
I cannot reproduce this. Please provide a video of this.

**Reply by agatemosu** [reply]
https://youtu.be/Z1ANCpS-bmw. `now playing` ignores the maps that are in the collection, it only serves to find a song faster

**Comment by Joehuu** [comment, marked as answer]
Already tracked in https://github.com/ppy/osu/issues/10507.
